spec/lib/checking_account_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe BankAccounts::CheckingAccount do
+
+  it "responds to #new" do
+    expect(BankAccounts::CheckingAccount).to respond_to :new
+  end
+
+  before :each do
+    @checking_account = BankAccounts::CheckingAccount.new("Name", 100)
+  end
```

Comment:
Rather than use an instance variable, you can use a `let` block to access objects set to variables. E.g., 
```
let(:checking_account) { BankAccounts::CheckingAccount.new("Name", 100) }
```